Sources/Client/Client/Client.swift

```diff
@@ -181,15 +181,10 @@ public final class Client: Uploader {
     ///   - config: the configuration object with details of how the new instance should be set up.
     ///   - defaultURLSessionConfiguration: the base URLSession configuration `Client` uses for its URL sessions.
     ///                                     `Client` is allowed to override the configuration with its own settings.
-    ///   - defaultWebSocketProviderType: the default WebSocket provider type. `Client` will create it on set user.
+    ///   - defaultWebSocketProviderType: the default WebSocket provider type. `Client` will create it on set user. `config.defaultWebSocketProviderType` will override this.
     init(config: Client.Config,
          defaultURLSessionConfiguration: URLSessionConfiguration = .default,
-         defaultWebSocketProviderType: WebSocketProvider.Type = {
-        if #available(iOS 13, *) {
-            return URLSessionWebSocketProvider.self
-        }
-        return StarscreamWebSocketProvider.self
-        }()) {
+         defaultWebSocketProviderType: WebSocketProvider.Type? = nil) {
```

Comment:
I'd remove this

Reply:
Needed for tests, unfortunately: to inject `MockWebSocketProvider`

Reply:
I see, thanks